src/main/scala/com/yotpo/metorikku/output/writers/file/HudiOutputWriter.scala

```diff
@@ -312,8 +328,61 @@ class HudiOutputWriter(props: Map[String, Object], hudiOutput: Option[Hudi]) ext
     df
   }
 
+  def hiveSyncManually(dataFrame: DataFrame, path: Option[String], hudiOutput: Option[Hudi]): DataFrame = {
+    var df = dataFrame
+    val ss = df.sparkSession
+    val catalog = ss.catalog
+
+    df = getHudiDf(df, ss, path)
+
+    val tablesToSync = getTablesToSyncByStorageType(hudiOutput, hudiOutputProperties.tableName.get)
+
+    for ((table, tableInputFormat) <- tablesToSync) {
```

Comment:
There is some shared logic between the tables.. no need to do it over and over again